tests/lifecycle/helper/helper.go

```diff
@@ -66,21 +66,6 @@ func DefinePod(name string) *corev1.Pod {
 		globalhelper.Configuration.General.TestImage)
 }
 
-// EnableMasterScheduling enables/disables master nodes scheduling.
-func EnableMasterScheduling(scheduleable bool) error {
```

Comment:
Why do we need to remove this function? We still use this function in lifecycle suite

Reply:
I didn't remove it, I moved it to a global level since I'm using it in this suite as well.